dotenv-sample

```diff
@@ -19,16 +19,14 @@ SENTRY_ENVIRONMENT=
 
 # Slack notifications
 SLACK_BOT_TOKEN=fake_token
-# URL to run jobs manually on job-server, used for the slack notification
-JOB_SERVER_JOBS_URL=https://jobs.opensafely.org/datalab/opensafely-interactive
 
-# Retrieve outputs from job-server
+# jobserver config
 JOB_SERVER_API=https://jobs.opensafely.org/
```

Comment:
I guess this should now be renamed `JOB_SERVER_URL` since it's used for accessing the website and the API

Reply:
Oh, yeah, the default meant it didn't error, will fix thanks